packages/cloud-cognitive/src/components/Datagrid/Datagrid/addons/RowSize/RowSizeRadioGroup.js

```diff
@@ -45,8 +45,8 @@ const RowSizeRadioGroup = ({
     <div
       className={`${blockClass}__row-size-dropdown`}
       style={{
-        top,
-        right,
+        top: top/16 + 'rem',
```

Comment:
Can you use the `rem()` function from `@carbon/layout`?

```
import { rem } from '@carbon/layout';

...
...
style={{
  top: rem(top),
  right: rem(right),
}}
```

Reply:
@matthewgallo changed to use `rem` carbon layout function.